isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1835,6 +1839,36 @@
             <item>no</item>
           </default>
         </parameter>
+
+        <parameter name="ADJUSTMENTOUT_H5">
+          <brief> Outputs bundle adjustment values to HDF5 file - adjustment_out.h5</brief>
+          <description>
+            Selection of this parameter flags output of bundle adjustment
+            values to a HDF5 file
+          </description>
+          <type>boolean</type>
+          <default>
+            <item>yes</item>
+          </default>
+        </parameter>
+
+        <parameter name="ADJUSTMENT_INPUT">
+          <type>filename</type>
+          <internalDefault>none</internalDefault>
+          <fileMode>input</fileMode>
+          <brief>
+            Reads in and applies adjustment values
+          </brief>
+          <description>
+            Reads in and applies bundle adjustment values from an HDF5 file,
+            usually the outputted adjustment_out.h5 file. Along with the three
+            required parameters, UPDATE must be set to YES for the input values 
+            to be read and applied.
+          </description>
+          <filter>
+            *.h5
+          </filter>
+        </parameter>
```

Comment:
I feel we should exclude other files if this is passed in? Things like the adjustment parameters and `ADJUSTMENTOUT_H5`. See line 1561 on this XML for an example. 

Slightly related, @amystamile-usgs has [the app XML docs](https://isis.astrogeology.usgs.gov/8.1.0/documents/HowToApplicationExamples/index.html) been transferred? I couldn't find it in the docs. If not it's fine it seems like it needs some work.

Reply:
Can you help specify which [params](https://isis.astrogeology.usgs.gov/8.2.0/Application/presentation/Tabbed/jigsaw/jigsaw.html) need to be excluded? I was thinking most since we're just applying the adjustment input values and no other calculations should/could be done during this anyway?

Reply:
yeaaaaahhh, it'll pretty much be everything that isn't the input list I think.